natgeo/mortar
# [SYTO-901] Add secondary nav to components page

https://jira.natgeo.com/browse/SYTO-901

To Test, 

Go to components page 
Verify it contains a left rail menu and that all links work.

Bonus:

- Fixed ESC key bug on site nav
